README.md

```diff
@@ -1,8 +1,10 @@
 # price_aggregator
-repository for learning how to do Agile and Scrum in CS362 and Oregon State
+repository for learning how to do Agile and Scrum in CS361 and Oregon State
 
 -Jessica Albert
 -Brandon Messenger
 -Kobe Chenea
 -Emily Edmondson
--Tanner Bahora
\ No newline at end of file
+-Tanner Bahora
+
+To run the server on your local machine, type command "python3 price_agg.py"
```

Comment:
This is correct for now, but @jmalbert7 might want to keep in mind she will need to change this line once the environment is setup.

Reply:
Will update run.sh to launch price_agg.py. Will also remove app.py and index.html from my PR

Reply:
@jmalbert7 looks like in Slack that @bamessenger is going to change it to app.py. Sorry for the confusion!

Reply:
#5 Provides instructions for launching the app by running the run.sh file. This comment will conflict with that instruction.